src/main/java/httpserver/http/route/RouteHandler.java

```diff
@@ -18,35 +20,20 @@ public String getResponse() {
         var requestSplitter = new RequestSplitter(request);
```

Comment:
The `RequestSplitter` is extracting data from the request, which is often referred to as "parsing", so could be a `RequestParser` which I think would be more idiomatic/abstract (i.e. splitting is what you're currently doing, but it's a how you're parsing the String to data). 

Again I would split the construction from the object being created - so there might be a method:
```
public static Request RequestParser.parse(String rawRequest)
```

With `Request` being a "POJO" with getter methods. Which would split the creation from the use.

Reply:
Thanks, Matt. I'll look into this